openthread/cmake/extensions.cmake

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+set(NRFXLIB_DIR ${ZEPHYR_NRFXLIB_MODULE_DIR})
+assert_exists(NRFXLIB_DIR)
```

Comment:
You are already in this repo.
Also be aware that this variable is not the one being used inside `openthread_libs_configuration_write()`, cause during execution of `openthread_libs_configuration_write()` then is solely depends on the `NRFXLIB_DIR` value in the scope of the caller.

Reply:
Thanks, fixed.